client/src/components/graph/graph.js

```diff
@@ -366,6 +365,29 @@ class Graph extends React.PureComponent {
       );
     }
 
+    const svg = stateChanges.centroids || centroids;
+
+    if (prevProps.pointDilation.categoryField !== pointDilation.categoryField) {
```

Comment:
As far as I can tell, this entire block just sets the `svg` value in state to undefined.   Is it dead code?

Reply:
dead code, removed